lib/cosmos/tools/tlm_viewer/widgets/labelvalue_widget.rb

```diff
@@ -20,20 +20,38 @@ class LabelvalueWidget < Qt::Widget
     include Widget
     include MultiWidget
 
-    def initialize(parent_layout, target_name, packet_name, item_name, value_type = :WITH_UNITS, characters = 12)
+    def initialize(parent_layout, target_name, packet_name, item_name, value_type = :WITH_UNITS, characters = 12, align = 'center')
       super(target_name, packet_name, item_name, value_type)
       setLayout(Qt::HBoxLayout.new)
       layout.setSpacing(0)
       layout.setContentsMargins(0,0,0,0)
+      layout.addStretch(1) if align.downcase == 'right'
```

Comment:
Ok, so 'right' means label and value will both be squished to the right, 'left' means the label and value will both be squished to the left, but 'center' will push the label to the left and the value to the right.  Is that actually the intent?  I was thinking this would only adjust the alignment of the label part of the label/value pair.  That is to say, the value would always be right-aligned, and the label would aligned within the remaining available space based on this keyword.

Assuming that is the behavior you're actually looking for, though, the changes look ok to me.